workshops/quotes_generator/README.md

```diff
@@ -0,0 +1,351 @@
+
+![image of directories in our project](https://cloud-9ne6wtuvp.vercel.app/0image.png)
+
+First, there are 2 main directories and a `package.json` file. We'll ignore the `package.json` file for now and let's have a look at the 2 directories, namely, `public/` and `src/`.
+
+Usually, the `public/` directory contains an HTML file and all your assets. We won't be touching the `public/` directory during the whole workshop, not even the HTML file!
+
+Next is the `src/` directory which contains all your JavaScript files and your CSS files. We have an `index.js` file, which basically renders our React code. The `style.css` file is already written for you.
+
+Next is the `App.js` file. Now basically, when we build big projects, we don't start directly with the `App.js` file. We create small components and then add them together in the `App` component. But today, we'll be building a quotes generator, which turns out to be pretty small and easy. So, we'll start writing our code directly in the `App` component to avoid making unnecessary components.
+
+This is what our `App.js` file looks right now.
+
+```jsx
+import React from "react";
+import "./styles.css";
+
+export default function App() {
+  return (
+    <div className="App">
+      <p>Hello World</p>
+    </div>
+  );
+}
+```
+
+This code simply renders out `Hello World` on your browser.
+
+Now we know where we are with our code, so it's time to start!
+
+### 2) The UI
+
+First let's build the basic UI of how our website should look like. Write the following code in the `App` component's `return()`.
+
+```jsx
+export default function App() {
+  return (
+    <div className="App">
+      <h1 className="title">Quotes Generator</h1>
+      <p className="quotes">This is a quote</p>
+      <p className="author">- author name</p>
+      <button className="button">New Quote</button>
+    </div>
+  );
+}
+```
+
+We built a very basic UI for our project and everything is self-explanatory here. The `button` doesn't right now and also we haven't yet made any `API calls`. So let's start adding the main functionality.
+
+This is what your preview window should look like:
+
+![Preview window](https://cloud-1ugc07d1z.vercel.app/0image.png)
+
+### 3) API Call
+
+Basically, we'll be making an API request to a url which will return us a json of quotes and their authors. Then we'll store this data in a state using the `useState` hook so we can re-render our component whenever it changes.
+
+![useState's epic picture](https://cloud-90fxjk0al.vercel.app/0image.png)
+
+The url where we will make the request is `https://type.fit/api/quotes`. If you open this link in the browser, you'll see a json file full of quotes!
+
+![What you see in the browser](https://cloud-h7wzq13xb.vercel.app/0image.png)
+
+Let's start working with it.
+
+First, we'll create a new variable to store this url, next we'll create an empty state to store the data.
+
+```jsx
+import React, { useState } from "react"; //<-- Import useState
```

Comment:
Wait, where should this code go? App.js? Because this is very different from the code that was previously in App.js. Clarify here